### PR TITLE
Adds ability to export to .reg file for use with PuTTY

### PR DESCRIPTION
**Pull request by conruggles**

### CODE REVIEW

**Comment by dylanaraps** [comment]
We can convert to RBG without the use of external processes/programs.

```sh
convert_to_rgb() {
    # Split the hex color and convert to RGB.
    red="$((16#${1:0:2}))"
    green="$((16#${1:2:2}))"
    blue="$((16#${1:4:2}))"

    printf "%s\n" "${red},${green},${blue}"
}
```

**Comment by conruggles** [comment]
I didn't realize how many commits I was racking up, I can rebase and open a
new PR tomorrow if you would like.

File: wal [hunk 1]
```diff
@@ -99,7 +99,7 @@ get_colors() {
 }
 
 
-# SET COLORSCHEME
+# SETi COLORSCHEME
```
Comment (dylanaraps) [inline wal]: `SETi` --> `SET`

File: wal [hunk 2]
```diff
@@ -328,10 +337,22 @@ reload_xrdb() {
...
+
 cleanup_pid () {
-  rm "$pid_file"
+    rm "$pid_file"
 }
 
+kill_old() {
+    # Kill old 'wal' instances.
+    pids=($(ps aux | awk '(!/ awk/) && /wal / {print $2}'))
+
```
Comment (dylanaraps) [inline wal]: Missing a closing `}`

**Comment by conruggles** [comment]
Alright didn't open a new PR, but all errors have been fixed.

File: wal [hunk 3]
```diff
@@ -110,6 +110,7 @@ set_color() {
     scss_colors+="\$color${1}: ${2};${newline}"
     firefox_colors+="  --color${1}: ${2};${newline}"
     plain+="${2}${newline}"
+    putty_colors+="\"Colour${1}\"=\"$(convert_to_rgb "${2}")\"${newline}"
```
Comment (dylanaraps) [inline wal]: Move this above `plain+=`.

File: wal [hunk 4]
```diff
@@ -298,6 +306,7 @@ export_colors() {
     export_rofi
     export_emacs
     export_plain     > "${cache_dir}/colors"
+    export_putty     > "${cache_dir}/wal.reg"
```
Comment (dylanaraps) [inline wal]: Can we name the file `colors.reg` or `colors-putty.reg` to match the other filenames?

**Comment by dylanaraps** [comment]
Sorry for all of the nitpicks :P

**Comment by conruggles** [comment]
Hey no problem! It's your project, I'm just happy that I can contribute.

**Comment by dylanaraps** [comment]
Thanks for the PR!

**Comment by dylanaraps** [comment]
Care to write some instructions for the README? Just a sentence or two explaining how to use the generated file. :)

**Comment by conruggles** [comment]
Yep, sure can!

**Pull request merged**